src/main/io/gps.c

```diff
@@ -1314,9 +1314,9 @@ void GPS_calculateDistanceAndDirectionToHome(void)
 }
 
 ////////////////////////////////////////////////////////////////////////////////////
-// Calculate the distance flown from gps position data
+// Calculate the distance flown and vertical speed from gps position data
 //
-static void GPS_calculateDistanceFlown(bool initialize)
+void GPS_calculateDistanceFlownVerticalSpeed(bool initialize)
```

Comment:
This should be `static`. And reordering the functions will get you out of having to put a forward declaration into the header file.

Reply:
TBH the fact that this function takes ```bool initialize``` as an argument is messy.

Reply:
@mikeller I was effectively avoiding to reorder the code, I will change it.
@stawiski I personally don't like the `bool initialize` as well, but it's preferred to prevent polluting the global namespace as I was told in other PRs.